### PR TITLE
Generate Raw ops from Tensorflow2.1.0rc1

### PR DESCRIPTION
**Pull request by Shashi456**
Fixes: #619
Execute has to be overloaded up to 14 variables due to the existence of `parseSequenceExampleV2`.

Updated `generatewrappers.swift` TensorFlow import command. 

High-arity return statement functions in `RawOpsGenerated.swift` have to be broken down by using let variables.

The accuracy of svd had to be changed for the tests to pass. 

Build and test passes.

### CODE REVIEW

File: Sources/TensorFlow/Bindings/EagerExecution.swift [hunk 1]
```diff
@@ -563,4 +563,243 @@ internal struct TFE_Op: TFTensorOperation {
         return result
     }
 
+    @inlinable @inline(__always)
```
Comment (dan-zheng) [inline Sources/TensorFlow/Bindings/EagerExecution.swift]: Please update `EagerExecution.swift.gyb` to generate up to arity 14 `execute` methods. Change [this line](https://github.com/tensorflow/swift-apis/blob/0f8bd19e812ee26f3d7cb68536172656b818138b/Sources/TensorFlow/Bindings/EagerExecution.swift.gyb#L311) to:
```
%for n in range(1, 14 + 1):
```

File: Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift [hunk 2]
```diff
@@ -74,7 +74,7 @@ final class LinearAlgebraTests: XCTestCase {
             }
             let aReconstituted = matmul(u!, matmul(s.diagonal(), 
                                         transposed: false, v!, transposed: true))
-            assertEqual(aReconstituted, a, accuracy: 1e-5)
+            assertEqual(aReconstituted, a, accuracy: 1e-3)
```
Comment (dan-zheng) [inline Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift]: > The accuracy of svd had to be changed for the tests to pass.

What error happens if you don't lower the accuracy? 1e-3 is a low accuracy value, which seems concerning - perhaps there's an implementation bug (mismatch between Swift and Python TensorFlow).

Comment (Shashi456) [reply Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift]: @dan-zheng /mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:77: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("2.3218951") is not equal to ("2.3218844") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:77: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("-3.1908154") is not equal to ("-3.1908028") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:89: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("2.3218951") is not equal to ("2.3218844") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:89: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("-3.1908154") is not equal to ("-3.1908028") +/- ("1e-05") -

The values seem very close to me. maybe i overdid it by decreasing it to 1e-3

Comment (dan-zheng) [reply Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift]: Thanks for sharing! Always using the tightest accuracy possible (`3e-5`) sounds great!

**Comment by Shashi456** [comment]
@dan-zheng is build time on kokoro really on the order of 30 minutes and more?

**Comment by dan-zheng** [comment]
> @dan-zheng is build time on kokoro really on the order of 30 minutes and more?

Could be. CI tests more than just [tensorflow/swift-apis](https://github.com/tensorflow/swift-apis), also [tensorflow/swift-models](https://github.com/tensorflow/swift-models) and other related repositories to ensure they're not broken.

**Comment by Shashi456** [comment]
@dan-zheng Changed the error bound and checked locally, the tests pass. i think this can be merged now

**Comment by dan-zheng** [comment]
> We should probably also change the commit hash in the tensorflow branch, since this is an api update?

Yep, feel free to submit a PR to [apple/swift](https://github.com/apple/swift/tree/tensorflow). Here's an example: https://github.com/apple/swift/pull/29079

**Pull request merged**